org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
+      <!-- Temperature scale:  'f' for Fahrenheit. 'c' for celsius -->
+      <arg name="scale" type="s" direction="in" />
```

Comment:
I don't feel strongly about the naming right now. At some point later we'll probably have a broader discussion about how to handle units in the schemas and translators, and we can update everything to be consistent then.

However the Wink thermostat will need to be updated if this parameter is added.

---
In reply to: [68283792](https://github.com/openT2T/translators/pull/22#discussion_r68283792) [](ancestors = 68283792,68283058)